### PR TITLE
Add support for AppVeyor

### PR DESCRIPTION
**Pull request by astrofrog**
This enables Windows continuous integration and fixes a few test failures.

### CODE REVIEW

**Comment by astrofrog** [comment]
The tests pass on Windows! :)

**Comment by astrofrog** [comment]
@ChrisBeaumont - any objections to merging this?

File: glue/core/tests/test_data.py [hunk 1]
```diff
@@ -38,7 +38,13 @@ def setup_method(self, method):
         self.comp_id = self.data.add_component(comp, 'Test Component')
 
     def test_2d_component_print(self):
-        assert str(self.comp) == 'Component with shape (2, 3)'
+        try:
+            assert str(self.comp) == 'Component with shape (2, 3)'
+        except AssertionError as e:  # Windows
+            try:
+                assert str(self.comp) == 'Component with shape (2L, 3L)'
```
Comment (ChrisBeaumont) [inline glue/core/tests/test_data.py]: minor nitpick: should we change the implementation instead of the test? "2L, 3L" seems potentially awkward

**Comment by ChrisBeaumont** [comment]
wow, super cool! +1 aside from note above

File: glue/core/tests/test_data.py (hunk 1 above)
Comment (astrofrog) [inline glue/core/tests/test_data.py]: Fixed

**Pull request merged**